src/com/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -288,9 +289,10 @@
     ;; confused if the database doesn't exist but we open and close a
     ;; connection without creating anything.
     (sql/with-connection write-db
-                         (scf-store/validate-database-version #(System/exit 1))
+                         (scf-store/validate-database-version
```

Comment:
Actually we already do this validation in the installer for PE whether you're running external or not.

Also I didn't see this was targeted for `2.3.x`, so nevermind about skipping the validation.